test/plugin/scenarios/postgresql-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,156 @@
+      peer: ''
+      peerId: 0
+      tags:
+      - {key: url, value: 'http://localhost:8080/postgresql-scenario/case/healthcheck'}
+      - {key: http.method, value: HEAD}
+  - segmentId: not null
+    spans:
+    -
+        operationName: PostgreSQL/JDBI/PreparedStatement/execute
+        operationId: eq 0
+        parentSpanId: 0
+        spanId: 1
+        tags:
+        - {key: "db.type", value: "sql"}
+        - {key: "db.instance", value: "postgres"}
+        - {key: "db.statement", value: "CREATE TABLE test_007(\nid VARCHAR(1) PRIMARY KEY, \nvalue VARCHAR(1) NOT NULL)"}
+        startTime: nq 0
+        endTime: nq 0
+        isError: false
+        spanLayer: Database
+        spanType: Exit
+        componentName: ''
+        componentId: 37
+        peer: postgresql-server:5432
+        peerId: eq 0
+    -
+        operationName: PostgreSQL/JDBI/CallableStatement/execute
+        operationId: eq 0
+        parentSpanId: 0
+        spanId: 2
+        tags:
+        - {key: "db.type", value: "sql"}
+        - {key: "db.instance", value: "postgres"}
+        - {key: "db.statement", value: "INSERT INTO test_007(id, value) VALUES(?,?)"}
+        startTime: nq 0
+        endTime: nq 0
+        isError: false
+        spanLayer: Database
+        spanType: Exit
+        componentName: ''
+        componentId: 37
+        peer: postgresql-server:5432
+        peerId: eq 0
+    -
+        operationName: PostgreSQL/JDBI/Statement/execute
+        operationId: eq 0
+        parentSpanId: 0
+        spanId: 3
+        tags:
+        - {key: "db.type", value: "sql"}
+        - {key: "db.instance", value: "postgres"}
+        - {key: "db.statement", value: "DROP table test_007"}
+        startTime: nq 0
+        endTime: nq 0
+        isError: false
+        spanLayer: Database
+        spanType: Exit
+        componentName: ''
+        componentId: 37
+        peer: postgresql-server:5432
+        peerId: eq 0
+    -
+        operationName: PostgreSQL/JDBI/Connection/close
+        operationId: eq 0
+        parentSpanId: 0
+        spanId: 4
+        tags:
+        - {key: "db.type", value: "sql"}
+        - {key: "db.instance", value: "postgres"}
+        - {key: "db.statement", value: ""}
+        startTime: nq 0
+        endTime: nq 0
+        isError: false
+        spanLayer: Database
+        spanType: Exit
+        componentName: ''
+        componentId: 37
+        peer: postgresql-server:5432
+        peerId: eq 0
+    -
+        operationName: PostgreSQL/JDBI/Connection/close
+        operationId: eq 0
+        parentSpanId: 0
+        spanId: 5
+        tags:
+        - {key: "db.type", value: "sql"}
+        - {key: "db.instance", value: "postgres"}
+        - {key: "db.statement", value: ""}
+        startTime: nq 0
+        endTime: nq 0
+        isError: false
+        spanLayer: Database
+        spanType: Exit
+        componentName: ''
+        componentId: 37
+        peer: postgresql-server:5432
+        peerId: eq 0
```

Comment:
seems `close` twice, why?

Reply:
ApmTest also this，may it‘s a bug，i will find it later